### PR TITLE
13.0 mig pos payment method image

### PR DESCRIPTION
**Pull request by hitrosol**
redo: pos_journal_image --> pos_payment_method_image, due to changing the model, etc.

### CODE REVIEW

File: pos_empty_home/static/src/js/db.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ odoo.define("pos_empty_home.db", function(require) {
 
     PosDB.include({
         get_product_by_category: function(category_id) {
-            if (category_id != 0) {
+            if (category_id !== 0) {
```
Comment (ivantodorovich) [inline pos_empty_home/static/src/js/db.js]: Please don't change other modules' files

File: pos_fix_search_limit/static/src/js/pos_fix_search_limit.js [hunk 2]
```diff
@@ -4,7 +4,8 @@ odoo.define("pos_fix_search_limit.db", function(require) {
     "use strict";
     var PosDB = require("point_of_sale.DB");
     PosDB.include({
-        limit: 314159265, // The maximum number of results returned by a search
+        limit: 314159265,
```
Comment (ivantodorovich) [inline pos_fix_search_limit/static/src/js/pos_fix_search_limit.js]: Please don't change other modules' files

File: pos_payment_method_image/models/pos_payment_method.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class PosPaymentMethod(models.Model):
+    _inherit = "pos.payment.method"
+
+    pos_image = fields.Binary(string="PoS Image", attachment=True)
```
Comment (ivantodorovich) [inline pos_payment_method_image/models/pos_payment_method.py]: I would use the new `fields.Image()` here

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
In general LGTM :+1: 

However:

- See my comments about changing other modules' files.
- Squash your migration commits into a single `[MIG] pos_payment_method_image : Migration to 13.0`

File: pos_payment_method_image/models/pos_payment_method.py (hunk 3 above)
Comment (legalsylvain) [reply pos_payment_method_image/models/pos_payment_method.py]: @ivantodorovich. is a specific migration required ? or the switch between both is implicit ?

Comment (ivantodorovich) [reply pos_payment_method_image/models/pos_payment_method.py]: No migration needed afaik it's extending the binary field with some optional resize tools

Doesn't add much value here but it's nice IMO 🤷🏻‍♂️

**Comment by hitrosol** [comment]
Thanks for the pointer @ivantodorovich, @legalsylvain and @chienandalu. The commit of "update translation files" is still red. I'm not sure how to fix it since it created by OCA Transbot. Can I update it manually ?

**Comment by legalsylvain** [comment]
1)  > The commit of "update translation files" is still red. 

Doesn't mind. the important is that the last commit is green.

2) Could you add a migration script as the model and the name changed ? (if you're not confident with migration script, please ping me)

**Comment by hitrosol** [comment]
@legalsylvain, this is my first commit, I'm not confident to do that now. Hopefully someone or you can make it ? I'll learn from this process for other commit. Thanks :)

**Comment by legalsylvain** [comment]
> Hopefully someone or you can make it ? 

No time for the time being. It is about openupgrade and openupgradelib features. 
I'll do it when I migrate my db from 12.0 to 16.0. 

In the meantime, could you just add a roadmap.rst file in readme folder to mention that migration from 12.0 is not covered ? 

thanks !

File: pos_fix_search_limit/static/src/js/pos_fix_search_limit.js [hunk 4]
```diff
@@ -15,7 +15,8 @@ odoo.define("pos_fix_search_limit.product_list", function(require) {
     var qweb = core.qweb;
 
     screens.ProductListWidget.include({
-        displayLimit: 100, // Number of elements displayed
+        displayLimit: 100,
+        // Number of elements displayed
```
Comment (ivantodorovich) [inline pos_fix_search_limit/static/src/js/pos_fix_search_limit.js]: this is still changing other module's files :)

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
apart from that, seems ok !

**Review by brendapaniagua — APPROVED** [review]
Good Work! LGTM!

**Comment by hitrosol** [comment]
I have followed up the items. Is there anything I missed ? Can we merge this ?

**Pull request closed without merging**